zerver/views/messages.py

```diff
@@ -336,7 +344,7 @@ def by_pm_with(self, query: Query, operand: str, maybe_negate: ConditionTransfor
             try:
                 narrow_profile = get_user_including_cross_realm(target_email, self.user_realm)
             except UserProfile.DoesNotExist:
-                raise BadNarrowOperator('unknown user ' + operand)
+                raise BadNarrowOperator('unknown user in operand')
```

Comment:
What's the story with these changes?  It's generally better to provide the object with the error, as we had before.

Reply:
I was getting this error when linting:
```
mypy      | zerver/views/messages.py:328: error: Unsupported operand types for + ("str" and "Iterable[int]")
mypy      | zerver/views/messages.py:328: note: Right operand is of type "Union[str, Iterable[int]]"
```

I figured out though that using `str(operand)` should fix this.